### ISSUE TITLE
Ne Exif Data

### ISSUE THREAD

**Issue by gine78** [Ultimatum22/MMM-RandomBackground#1]
Hello,

i receive no exif data. The value is 00:00:00 00:00:00 all the time. 

has anyone a solution for this?

**Comment by Ultimatum22** [comment]
That's correct, for now the Exif data is not working. I am considering to take it since it takes a long time to process an image, which presents an issue when loading a lot of images.

**Comment by djtale** [comment]
Hello, I have also sometimes that error in my log file :
No Exif segment found in the given image.

In node_helper.js, I tried to add some debug information :
`if (error) {
   console.log('Error: for photo ' + i + ' - ' + data[i] + ' - ' + error.message);
} else {
   console.log('Exif found for photo ' + i + ' - ' + data[i] + ' - ' + dateTaken);`

The problem is that the increment i doesn't seem to be working. As a result I get those messages :
[2020-05-15 09:01:19.482] [LOG]    Error: for photo 86 - undefined - No Exif segment found in the given image.
[2020-05-15 09:01:19.483] [LOG]    Error: for photo 86 - undefined - No Exif segment found in the given image.
[2020-05-15 09:01:19.561] [LOG]    Exif found for photo 86 - undefined - undefined
[2020-05-15 09:01:19.638] [LOG]    Exif found for photo 86 - undefined - 2018:05:23 12:09:38
[2020-05-15 09:01:19.704] [LOG]    Exif found for photo 86 - undefined - 2019:07:08 09:55:01

I don't understand why the value for i is always 86 (it is the number of images in my folder).
